### PR TITLE
Kachin Keyboard + Dictionary 

### PR DESCRIPTION
**Pull request by kokoye2007**
Add Kachin Dictionary [US+ Internation] base

* https://en.wikipedia.org/wiki/Kachin_State

* https://en.wikipedia.org/wiki/Kachin_people 

* https://en.wikipedia.org/wiki/Jingpo_people

* https://en.wikipedia.org/wiki/Jingpho_language

### CODE REVIEW

**Comment by martholomew** [comment]
This is a great PR! However, you're going to need to change the `defaultDictionaryLocale` in `kachin/pack/src/main/res/xml/kachin_keyboards.xml` for each keyboard to `kac`, otherwise it will, I believe, use the English dictionary. Also, I think it would be a good idea to `cat` all of the files in `kachin/pack/dictionary/inputs/` together, just to avoid that excess of files.

**Comment by menny** [comment]
@martholomew I've invited you to the AnySoftKeyboard team. If you accept, you will be able to review and merge PRs.

**Comment by martholomew** [comment]
@menny Awesome, thank you very much! Thanks for the fixes @kokoye2007 !

File: settings.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ include ':languages:brazilian:pack', ':languages:brazilian:apk'
 include ":languages:catalan:pack", ":languages:catalan:apk"
 include ":languages:czech:pack", ":languages:czech:apk"
 include ':languages:english:pack', ':languages:english:apk'
+include ':languages:kachin:pack', ':languages:kachin:apk'
```
Comment (martholomew) [inline settings.gradle]: Sorry I missed this, but you only need this line once, if you would be able to change that.

**Comment by menny** [comment]
thank you!

**Pull request merged**